marathon/datadog_checks/marathon/marathon.py

```diff
@@ -167,22 +176,23 @@ def process_deployments(self, url, timeout, auth, acs_url, ssl_verify, tags=None
         if response is not None:
             self.gauge('marathon.deployments', len(response), tags=tags)
 
-    def process_queues(self, url, timeout, auth, acs_url, ssl_verify, tags=None):
-        # Number of queued applications
-        QUEUE_PREFIX = 'marathon.queue'
+    def process_queues(self, url, timeout, auth, acs_url, ssl_verify, tags=None, group=None):
```

Comment:
I added `group` since I needed to ultimately pass it to `get_apps_json` and I didn't want to make it so that `process_apps` had to be called before `process_queues` to set `self.apps_response` in order to work correctly since that would not be obvious.